application/common/config/main.php

```diff
@@ -94,6 +94,7 @@
             'subjectForMfaOptionRemoved' => Env::get('SUBJECT_FOR_MFA_OPTION_REMOVED'),
             'subjectForMfaEnabled' => Env::get('SUBJECT_FOR_MFA_ENABLED'),
             'subjectForMfaDisabled' => Env::get('SUBJECT_FOR_MFA_DISABLED'),
+            'subjectForMethodVerify' => Env::get('SUBJECT_FOR_METHOD_VERIFICATION'),
```

Comment:
For naming consistency, what would you think about changing `SUBJECT_FOR_METHOD_VERIFICATION` to `SUBJECT_FOR_METHOD_VERIFIFY`?